inc/vcf/validator.hpp

```diff
@@ -173,7 +175,8 @@ namespace ebi
     using QuickValidator_v43 = ParserImpl_v43<QuickValidatorCfg>;
     using FullValidator_v43 = ParserImpl_v43<FullValidatorCfg>;
     using Reader_v43 = ParserImpl_v43<ReaderCfg>;
-    
+
+    std::unique_ptr<ebi::vcf::Parser> build_parser(std::string const & path, ValidationLevel level, ebi::vcf::Version version);
```

Comment:
`ebi::vcf` qualifiers shouldn't be necessary since they are already in the same namespace.